LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan/Data128.cs

```diff
@@ -39,8 +39,8 @@ public override string ToString()
 
         public static Data128 Read(ReadOnlySpan<byte> buffer)
         {
-            var lo = BinaryPrimitives.ReadUInt64BigEndian(buffer);
-            var hi = BinaryPrimitives.ReadUInt64BigEndian(buffer[8..]);
+            var hi = BinaryPrimitives.ReadUInt64BigEndian(buffer);
```

Comment:
can we add unit tests, I don't see coverage for this?

Reply:
There is a unit test for this, added it in b0701987576f4c9f723053cbbbaa7b8dc34cedfb

Reply:
that's not unit testing Data128, is it? I would assume to have unit tests for the implementation of that type?

Reply:
As of now we don't have any unit tests of `Data128` at all, it's all implicitly covered through the tests of the different key types. I will add that test for `Data128` there, but I won't write the tests for the existing functionality if that's OK

Reply:
Done in 5d3a17d6dbf5c7b706164cd525c16f21aed01c50